src/search_algo.hpp

```diff
@@ -497,15 +497,45 @@ seedLooksPromising(LocalDataHolder<TGlobalHolder> const & lH,
     return false;
 }
 
+/*** THIS ACCESSES seqan3::detail:: IT SHOULD BE REPLACED WITH OFFICIAL INTERFACE ONCE OPTIMISED ***/
```

Comment:
Do I understand it correctly that we use the detail functions to enable adaptive seeding or is there a second problem where the general search is too slow (and why)?

Reply:
This is entirely unrelated, adaptive seeding happens only via
https://github.com/seqan/lambda/pull/153/commits/c241387b97808d011c531d3ef70c6313144c1972#diff-ee6754a210a5bb744bff645ba899ae8dR570

The function you marked circumenvents various unecessary memory allocations in `seqan3::search`. See https://github.com/seqan/seqan3/pull/1528 for more details.

Reply:
Ah thanks for clarifying this.